test/e2e/sched/sched_test.go

```diff
@@ -204,5 +209,102 @@ var _ = Describe("[Scheduler] imageReplacement", func() {
 				return true
 			}).WithTimeout(time.Minute * 2).WithPolling(time.Second * 30).Should(BeTrue())
 		})
+		It("should reflect changes in cacheResyncPeriod when configured", func() {
+			deployment, err := podlist.With(e2eclient.Client).DeploymentByOwnerReference(context.TODO(), nroSchedObj.UID)
```

Comment:
for the future: let's read the namespace/name pair from `NUMAResourcesScheduler.Statua`